src/NServiceBus.TransportTests/When_on_error_invoked.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace NServiceBus.TransportTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using Transport;
+    using NUnit.Framework;
+
+    public class When_on_error_invoked : NServiceBusTransportTest
+    {
+        [TestCase(TransportTransactionMode.None)]
+        [TestCase(TransportTransactionMode.ReceiveOnly)]
+        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
+        [TestCase(TransportTransactionMode.TransactionScope)]
+        public async Task Should_share_extensions_context_with_message_context(TransportTransactionMode transactionMode)
```

Comment:
Isn't https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.TransportTests/When_setting_context_items.cs covering this scenario?

Reply:
yes, I think that test covers that 👍 thanks, seems like we didn't see that test